include/git2/worktree.h

```diff
@@ -137,23 +161,46 @@ typedef enum {
 	GIT_WORKTREE_PRUNE_WORKING_TREE = 1u << 2,
 } git_worktree_prune_t;
 
+typedef struct git_worktree_prune_options {
+	unsigned int version;
+
+	char prune_valid; /**< prune even if working tree is valid */
+	char prune_locked; /**< prune even if working tree is locked */
+	char delete_worktree; /**< delete complete working directory */
```

Comment:
Do we use a `char` anywhere else to represent a boolean value within a struct?  I feel like the common pattern for our public API would be to have a `flags` value in the options struct...  for example, in [`diff`](https://github.com/libgit2/libgit2/blob/master/include/git2/diff.h#L290).

(Internally we use `int` bitfields in places, but we don't do that in the public API.)